x-pack/plugins/beats_management/public/config_schemas.ts

```diff
@@ -4,38 +4,53 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { i18n } from '@kbn/i18n';
 import { YamlConfigSchema } from './lib/lib';
 
 const filebeatInputConfig: YamlConfigSchema[] = [
   {
     id: 'paths',
     ui: {
-      label: 'Paths',
+      label: i18n.translate('xpack.beatsManagement.filebeatInputConfig.pathsLabel', {
```

Comment:
This file needs to remain a plane JS object, the parsing needs to take place at the consumption point. The reason being, this file will be replaced in the next version with JSON stored in ES

Reply:
That’s perfect!